helpers/hstring.py

```diff
@@ -94,6 +94,48 @@ def get_docstring_line_indices(lines: List[str]) -> List[int]:
     return docstring_line_indices
 
 
+# TODO(allenmatt10): Add tests.
+def get_function_indices(lines: List[str]) -> List[int]:
+    """
+    Get indices of lines of code that are inside functions and methods.
+
+    :param lines: the code lines to check
+    :return: the indices of function lines
+    """
+    function_line_indices = []
+    i = 0
+    n = len(lines)
+    while i < n:
+        # Process each line to find function header.
+        line = lines[i]
+        match = re.match(r"(\s*)def\s+\w+", line)
```

Comment:
When there are regex let's put a comment with an example of what we are looking for